cypress/integration/footerSpec.js

```diff
@@ -26,9 +26,12 @@ describe('Footer Tests', () => {
   it('should contain copyright text', () => {
     cy.get('footer p').should(
       'contain',
-      `© ${new Date().getFullYear()} BBC. The BBC is not responsible for the content of external sites. `,
+      `${new Date().getFullYear()} BBC. The BBC is not responsible for the content of external sites.`,
     );
   });
+  it('copyright symbol should have language attribute', () => {
```

Comment:
This should be checking if the `lang` attribute is set, not if it has the `copyright` symbol. The test description is misleading.